pkg/pgmodel/insert_ctx.go

```diff
@@ -23,7 +23,7 @@ func FinishWriteRequest(wr *prompb.WriteRequest) {
 			ts.Labels[j] = prompb.Label{}
 		}
 		ts.Labels = ts.Labels[:0]
-		ts.Samples = ts.Samples[:0]
+		ts.Samples = nil
```

Comment:
wonder if this log should be in parseData instead?

Reply:
mmm, both

Reply:
ehhh, fine, just in `parseData`